src/client_backend/client_backend.h

```diff
@@ -652,8 +652,7 @@ class InferResult {
   /// \return Error object indicating the success or failure of the
```

Comment:
I'm getting a build error on a clean rebuild:

```
In file included from /root/dev/perf_analyzer/src/client_backend/torchserve/torchserve_client_backend.cc:27:
/root/dev/perf_analyzer/src/client_backend/torchserve/torchserve_client_backend.h:104:9: error: 'triton::perfanalyzer::clientbackend::Error triton::perfanalyzer::clientbackend::torchserve::TorchServeInferResult::RawData(const string&, const uint8_t**, size_t*) const' marked 'override', but does not override
  104 |   Error RawData(
      |         ^~~~~~~
```

Reply:
Let me take a look, looks like some of the changes were not saved

Reply:
Yeah, fixed it, the build was successful in the latest pipeline.